vent/sqldb/sqldb.go

```diff
@@ -18,15 +18,17 @@ type SQLDB struct {
 	DB        *sql.DB
 	DBAdapter adapters.DBAdapter
 	Schema    string
+	ChainID   string
 	Log       *logger.Logger
 }
 
 // NewSQLDB delegates work to a specific database adapter implementation,
 // opens database connection and create log tables
 func NewSQLDB(connection types.SQLConnection) (*SQLDB, error) {
 	db := &SQLDB{
-		Schema: connection.DBSchema,
-		Log:    connection.Log,
+		Schema:  connection.DBSchema,
+		ChainID: connection.ChainID,
```

Comment:
Let's pull this and `BurrowVersion` up to `SQLDB` as the 'SQL Burrow context fields'